cli/tests/pcluster_config/test_pcluster_config_convert/test_slurm_sit_full/pcluster.config.ini

```diff
@@ -63,11 +63,13 @@ fsx_fs_id = fs-0aaae053900f84047
 shared_dir = sharedebs
 volume_type = gp2
 volume_size = 50
+volume_iops = 100
 
 [ebs settings2]
 shared_dir = sharedebs2
 volume_type = gp2
 volume_size = 10
+volume_iops = 100
```

Comment:
let's remove this from here so that we also test with an unset value for iops. Let's make sure that when converting a config without iops set we don't get 100 in the resulting converted config.

Actually gp2 should not have iops at all, so if we want the test to be semantically correct let's switch one of to gp3 and remove iops from the gp2.

Reply:
Done!